index.bs

```diff
@@ -58,6 +58,7 @@ spec:webidl; type:interface; text:Promise
 <link rel="preload" href="https://www.w3.org/scripts/MathJax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js?rev=2.7.5" as="script">
 <link rel="preload" href="https://www.w3.org/scripts/MathJax/2.7.5/jax/element/mml/optable/BasicLatin.js?rev=2.7.5" as="script">
 <link rel="preload" href="https://www.w3.org/scripts/MathJax/2.7.5/jax/output/HTML-CSS/autoload/mtable.js?rev=2.7.5" as="script">
+<script src="//www.w3.org/scripts/TR/2016/fixup.js" type="application/javascript"></script>
```

Comment:
Is this the right place to get fixup.js?

Reply:
Just curious - why do we need fixup.js? Can you point me to the discussion?